public/API/API_GetUserCompletionProgress.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/*
+ *  API_GetUserCompletionProgress - gets the entire completion progress (eg: /user/{username}/progress) for a user
+ *                                  similar to `GetUserCompletedGames`, but only includes a single record for each game
+ *                                  and also includes the game's current award level as shown on the "Completion Progress" page.
+ *    u : username
+ *    o : offset - number of entries to skip (default: 0)
+ *    c : count - number of entries to return (default: 100, max: 500)
```

Comment:
The feature description says the max is 100.
> Pagination is supported with o and c parameters. A max of 100 games can be returned at once, similar to the Completion Progress page itself.

Reply:
I think some wires got crossed in my brain.

However, I'm wondering if it's okay for the endpoint to have a 500 max limit (the current implementation). Looks like the response size is ~134KB.

Reply:
Given that we're fetching the entire list before sorting, the limit only applies to the bandwidth, not the query or post-processing. This is probably fine.